Kitodo/src/main/webapp/WEB-INF/templates/includes/userEdit/removeClientFromUserDialog.xhtml

```diff
@@ -0,0 +1,70 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+        xmlns="http://www.w3.org/1999/xhtml"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:f="http://xmlns.jcp.org/jsf/core"
+        xmlns:p="http://primefaces.org/ui">
+    <p:dialog class="remove-client-dialog"
+              id="removeClientDialog"
+              widgetVar="removeClientDialog"
+              showHeader="false"
+              modal="true"
+              width="500px"
+              appendTo="@(body)"
+              resizable="false"
+              closable="false">
+        <h3>#{msgs.removeClientProjectsFromUser}</h3>
+        <h:form id="removeFromProjectsForm">
+            <div class="select-note ui-messages-info">#{msgs.removeClientProjectsFromUserConfirmMessage}</div>
+            <div class="select-selector">
+                <p>
+                    <p:dataTable id="clientProjects"
+                                 style="max-height: 100px; overflow-y: scroll; border: 1px solid var(--blue); border-radius: 3px;"
```

Comment:
Suggestion: For reusing and browser caching create css class, move style to kitodo.css and use styleClass instead or use global style if exists.